Mogakco/Sources/Presentation/Common/View/AnimationView.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  AnimationView.swift
+//  Mogakco
+//
+//  Created by 이주훈 on 2022/11/30.
+//  Copyright © 2022 Mogakco. All rights reserved.
+//
+
+import UIKit
+
+import SnapKit
+
+final class AnimationView: UIView {
+    
+    enum Animation {
+        static let iconCount: Int = 5
+        static let IconSize: Int = 100
```

Comment:
```suggestion
        static let iconSize: Int = 100
```
변수명이 대문자로 시작하고 있습니다!

Reply:
오 감사합니다 수정해 놓을게요!